ui/app/routes/jobs/job/versions.js

```diff
@@ -15,21 +15,8 @@ import { inject as service } from '@ember/service';
 export default class VersionsRoute extends Route.extend(WithWatchers) {
   @service store;
 
-  queryParams = {
-    diffVersion: {
-      refreshModel: true,
-    },
-  };
-
```

Comment:
Now that the route doesn't care about Diff data (the controller does), we don't need to make it concerned with queryParams / it doesn't need to re-fetch the versions when the user switches the comparison version via dropdown